pkg/scheduler/apis/config/v1beta1/conversion.go

```diff
@@ -73,6 +142,9 @@ func convertToInternalPluginConfigArgs(out *config.KubeSchedulerConfiguration) e
 				return fmt.Errorf("converting .Profiles[%d].PluginConfig[%d].Args into internal type: %w", i, j, err)
 			}
 			out.Profiles[i].PluginConfig[j].Args = internalArgs
+			if !strings.HasPrefix(out.Profiles[i].PluginConfig[j].Name, framework.IntreePluginPrefix) {
```

Comment:
Shouldn't you use `intreePluginNamesWithoutPrefixSet` here too?

Add a test for unknown plugin names not getting converted.

Reply:
Use `intreePluginNamesWithoutPrefixSet` here too